deploy.sh

```diff
@@ -235,6 +237,27 @@ function cleanup_helm_data {
   kc -n kube-system delete cm -l OWNER=TILLER 2> /dev/null || true
 }
 
+function cleanup_iot_devices {
```

Comment:
should we add an early return, a la
gcloud services list --project="${GC_PROJECT_ID}" | grep -q cloudiot.googleapis.com || return

Reply:
The service list is probably slower than the quick fail of the next command :)

Reply:
I will test it